pod.go

```diff
@@ -72,6 +72,9 @@ type State struct {
 
 	// Bool to indicate if the drive for a container was hotplugged.
 	HotpluggedDrive bool `json:"hotpluggedDrive"`
+
+	// HypervisorState keeps the state of the hypervisor
+	HypervisorState HypervisorState `json:"hypervisorState"`
```

Comment:
This is not the way we have implemented the state files on virtcontainers. Let me think about it, I'll try to come up with a clean solution for this storage purpose (now that I am clear about what we expect).
@amshinde please take a look and give some inputs about this too.